[2024-08-14T01:32:59.559+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2024-08-13T00:00:00+00:00 [queued]>
[2024-08-14T01:32:59.568+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2024-08-13T00:00:00+00:00 [queued]>
[2024-08-14T01:32:59.568+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-08-14T01:32:59.569+0000] {taskinstance.py:1363} INFO - Starting attempt 2 of 2
[2024-08-14T01:32:59.570+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-08-14T01:32:59.581+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2024-08-13 00:00:00+00:00
[2024-08-14T01:32:59.588+0000] {standard_task_runner.py:55} INFO - Started process 18804 to run task
[2024-08-14T01:32:59.592+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'scheduled__2024-08-13T00:00:00+00:00', '--job-id', '78', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpobygu75l']
[2024-08-14T01:32:59.595+0000] {standard_task_runner.py:83} INFO - Job 78: Subtask create_table
[2024-08-14T01:32:59.664+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table scheduled__2024-08-13T00:00:00+00:00 [running]> on host a76316aa0937
[2024-08-14T01:32:59.729+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2024-08-13T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-08-13T00:00:00+00:00
[2024-08-14T01:32:59.741+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2024-08-14T01:32:59.748+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users(
            firstname  TEXT NOT NULL,
            lastname TEXT NOT NULL,
            username TEXT NOT NULL,
            password TEXT NOT NULL,
            email TEXT NOT NULL
            );
        , parameters: None
[2024-08-14T01:32:59.749+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2024-08-14T01:32:59.760+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20240813T000000, start_date=20240814T013259, end_date=20240814T013259
[2024-08-14T01:32:59.807+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2024-08-14T01:32:59.838+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
